src/common/fam_internal_exception.h

```diff
@@ -151,34 +148,33 @@ inline enum Fam_Error convert_to_famerror(enum Internal_Error serverErr) {
     case LIBFABRIC_ERROR:
         return FAM_ERR_LIBFABRIC;
 
-    case ALLOC_NO_ERROR:
     case REGION_NOT_INSERTED:
     case DATAITEM_NOT_INSERTED:
     case REGION_NOT_REMOVED:
     case DATAITEM_NOT_REMOVED:
-    case REGION_NOT_MODIFIED:
+    case REGION_NAME_TOO_LONG:
+    case DATAITEM_NAME_TOO_LONG:
```

Comment:
i think we should return FAM_ERR_NAME_TOO_LONG  in case of REGION_NAME_TOO_LONG or DATAITEM_NAME_TOO_LONG